docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.5'
+services:
+  xblock_sdk:
+    image: xblock-sdk
```

Comment:
Optional: Can add `container_name: edx.devstack.xblock-sdk` and then use it instead of the auto-generated name `xblock-sdk_xblock_sdk_1 `.

Reply:
Done